packages/subgraph/src/mappings/cfav1.ts

```diff
@@ -177,12 +228,26 @@ export function handleStreamUpdated(event: FlowUpdated): void {
         return;
     }
 
-    let stream = getOrInitStream(
-        senderAddress,
-        receiverAddress,
-        tokenAddress,
-        event.block
+    // create or update stream entity
+    let cfaContract = IConstantFlowAgreementV1.bind(event.address);
+    let depositResult = cfaContract.try_getFlow(
```

Comment:
I'm very skeptical of introducing this to the mapper of the most common event in our protocol...

@hellwolf

Can we benchmark the effect of this?

Reply:
will add a checker for this (previous blocks will have to try_getFlow)

Reply:
will be sure to do this in the future